<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[PASS: The iteration number is less then 5. it's 0]]>
    </line>
    <line>
      <![CDATA[PASS: The iteration number is less then 5. it's 0]]>
    </line>
    <line>
      <![CDATA[PASS: The iteration number is less then 5. it's 1]]>
    </line>
    <line>
      <![CDATA[PASS: The iteration number is less then 5. it's 1]]>
    </line>
    <line>
      <![CDATA[PASS: The iteration number is less then 5. it's 2]]>
    </line>
    <line>
      <![CDATA[PASS: The iteration number is less then 5. it's 2]]>
    </line>
    <line>
      <![CDATA[PASS: The iteration number is less then 5. it's 3]]>
    </line>
    <line>
      <![CDATA[PASS: The iteration number is less then 5. it's 3]]>
    </line>
    <line>
      <![CDATA[PASS: The iteration number is less then 5. it's 4]]>
    </line>
    <line>
      <![CDATA[PASS: The iteration number is less then 5. it's 4]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="10109" started-at="2017-08-24T10:46:26Z" finished-at="2017-08-24T10:46:36Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="10109" started-at="2017-08-24T10:46:26Z" finished-at="2017-08-24T10:46:36Z">
      <class name="Tests.TwoThreadsTest">
        <test-method status="FAIL" signature="test()[pri:0, instance:Tests.TwoThreadsTest@55f3ddb1]" name="test" duration-ms="10070" started-at="2017-08-24T13:46:26Z" finished-at="2017-08-24T13:46:36Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Test failed with failures:
The iteration number is less then 5. it's 5
The iteration number is less then 5. it's 6
The iteration number is less then 5. it's 7
The iteration number is less then 5. it's 8
The iteration number is less then 5. it's 9
The iteration number is less then 5. it's 5
The iteration number is less then 5. it's 6
The iteration number is less then 5. it's 7
The iteration number is less then 5. it's 8
The iteration number is less then 5. it's 9]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Test failed with failures:
The iteration number is less then 5. it's 5
The iteration number is less then 5. it's 6
The iteration number is less then 5. it's 7
The iteration number is less then 5. it's 8
The iteration number is less then 5. it's 9
The iteration number is less then 5. it's 5
The iteration number is less then 5. it's 6
The iteration number is less then 5. it's 7
The iteration number is less then 5. it's 8
The iteration number is less then 5. it's 9
	at org.testng.Assert.fail(Assert.java:93)
	at Infra.ReporterTest.endTest(ReporterTest.java:94)
	at Tests.TwoThreadsTest.test(TwoThreadsTest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[PASS: The iteration number is less then 5. it's 0]]>
            </line>
            <line>
              <![CDATA[PASS: The iteration number is less then 5. it's 0]]>
            </line>
            <line>
              <![CDATA[PASS: The iteration number is less then 5. it's 1]]>
            </line>
            <line>
              <![CDATA[PASS: The iteration number is less then 5. it's 1]]>
            </line>
            <line>
              <![CDATA[PASS: The iteration number is less then 5. it's 2]]>
            </line>
            <line>
              <![CDATA[PASS: The iteration number is less then 5. it's 2]]>
            </line>
            <line>
              <![CDATA[PASS: The iteration number is less then 5. it's 3]]>
            </line>
            <line>
              <![CDATA[PASS: The iteration number is less then 5. it's 3]]>
            </line>
            <line>
              <![CDATA[PASS: The iteration number is less then 5. it's 4]]>
            </line>
            <line>
              <![CDATA[PASS: The iteration number is less then 5. it's 4]]>
            </line>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- Tests.TwoThreadsTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
